docs/guides/software-lifecycle/security/security-scanning/README.md

```diff
@@ -0,0 +1,208 @@
+     ```
+
+3. **Scanning**
+   - For more information about running SCRUB, refer to the [user documentation](https://nasa.github.io/scrub/usage.html)
+
+    ```bash
+    scrub run
+    ```
+
+4. **Checking Results**
+   - Review the `<tool>.scrub` file to audit any reported security issues:
+
+      ```bash
+      vi .scrub/<tool>.scrub
+      ```
+
+[View more on advanced SCRUB scan configuration](https://nasa.github.io/scrub/configuration-inputs.html)
+
+> ℹ️ **Note**: Any confirmed security issues should be addressed and mitigated before pushing to remote repositories.
+
+### GitHub.com Actions Analysis on Push and Pull Request
+
+Code is scanned for security risks within the repository. It leverages [GitHub Action](https://github.com/features/actions). The scan is triggered during a push or pull request and any detected security vulnerabilities are reported while blocking merges or pushes to protected branches.
+
+#### Steps
+1. **Workflow Creation**
+   - The first step is to create a `codeql.yaml` workflow file in the `.github/workflows` directory to define the GitHub action. Copy and paste the below to your new file while ensuring the correct branch of your codebase is referenced. For example, the following configuration scans for CodeQL security and quality checks for Python language code. Note: a version of the below is also available through the [SLIM Python Starter Kit](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/workflows/codeql.yml):
+   - This workflow is based on the default CodeQL workflow file with three modifications:
+      1. Under the *Initialize CodeQL* step, the `queries` entity has been added to enable all of the available security queries
+      2. A new *Post-Process Output* step has been added to generate a CSV output file that may be easily ingested by other systems
+      3. A new *Upload CodeQL Artifacts* step has been added to produce a set of archive files for each run
+
+```yaml
+name: "CodeQL"
+
+on:
+  push:
+    branches: [main, develop]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [develop]
+  schedule:
+    # default branch on sundays at 5a
+    - cron: '0 5 * * 0'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # CodeQL supports ['cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby']
+        # Learn more about CodeQL language support at https://git.io/codeql-language-support
+        language: ['python']
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v3
+      with:
+        #config-file: ./.github/workflows/codeql/codeql-config.yml
```

Comment:
Why is `config-file` commented-out here? I like the practice of always including a comment whenever something is commented-out—and if nothing appropriate comes to mind, then just delete the line!

Reply:
I've added  another comment for context.